### ISSUE TITLE
[TODO] Game Board: Counter Phase

### ISSUE THREAD

**Issue by Bratah123** [BAA-Studios/MOOgiwara#42]
## Abstract
The Counter Phase is the phase during an attack. When a player announces an attack on another card, the other player is allowed to play counter cards out of their hand in response to the attack.

## Details
If the counter cards all equate to a number greater than the number the player is attacking with, the block is successful.
Players are allowed to play more than one counter card during their counter phase out of their hand.

**Comment by BBeretta** [comment]
Hey,
In addition to the Counter of the cards (Counter Phase), it also has the possibility of activating **Events** Cards.

**Comment by Bratah123** [comment]
> Hey, In addition to the Counter of the cards (Counter Phase), it also has the possibility of activating **Events** Cards.

Yup! we would consider the counter event cards as counter cards aswell

**Comment by Bratah123** [comment]
Preview/Demo of the counter stack:

You can click hide to hide the board.

https://user-images.githubusercontent.com/58405975/221970751-a201f620-d988-48ed-861c-83876f42ac98.mp4
